contact.html

```diff
@@ -1,11 +1,86 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Contact</title>
-</head>
-<body>
-  
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="contacts.css">
+    <script type="module" src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.esm.js" defer></script>
+    <script nomodule src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.js" defer></script>
+    <script src="/form-handler.js" defer></script>
+    <title>Contact</title>
+  </head>
+  <body>
+    <header class="header">
+      <nav>
+        <ul class="navigation-list">
+          <li>
+            <a href="/home.html">Home</a>
+          </li>
+          <li>
+            <a href="/contact.html">
+              Contact
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <h1>Contact Me</h1>
+
+      <!-- Додаємо ID для форми -->
+      <form class="form" id="contact-form" inputmode="contact-form" method="post">
+        <section>
+          <div class="name">
+            <label for="full-name">Full Name*</label>
+
+            <input type="text" placeholder="Your first name" id="first-name" name="first-name" required>
+            <input type="text" placeholder="Your last name" id="last-name" name="last-name" required>
+          </div>
```

Comment:
You do not have an input with id "full-name", so "for" attribute for label does not work